platform/ios/src/MGLMapboxEvents.h

```diff
@@ -66,7 +66,10 @@ typedef NS_MUTABLE_DICTIONARY_OF(NSString *, id) MGLMutableMapboxEventAttributes
 + (void) flush;
 
 // Main thread only
-+ (void)validate;
++ (void) validate;
+
+// Main thread only
++ (void) ensureMetricsOptoutExists;
```

Comment:
We should be asserting that we’re on the main thread, using `MGLAssertIsMainThread()`, instead of using these Java annotation–like comments, which mean nothing.